### ISSUE TITLE
Design Intent - Accessibility Feedback - Authenticated Experience, Contact Information, Email Verification

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#108764]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** As you refine your design, make sure you're materially honest around buttons and links --- if it navigates the user somewhere it's coded as a link and is visually identifiable as a link, if it performs an action on the page or submits user data it's coded as a button and is visually identifiable as a button. Buttons and links can have different interaction patters in a variety of assistive technologies, and you want to make sure all users are able to (1) identify what kind of element it is, and (2) have it behave in a way that matches their expected interaction pattern.
   
   In particular, I'm thinking about [the CTA in the email](https://www.figma.com/design/bFdl7MEIda4ExZIQuot84r/Profile---Contact-Information?node-id=1755-10340&t=XSXf9ftEhGAIFAtm-4) and the [verify/re-verify/re-send actions in each warning alert](https://www.figma.com/design/bFdl7MEIda4ExZIQuot84r/Profile---Contact-Information?node-id=1757-14416&t=XSXf9ftEhGAIFAtm-4).
- [ ] **Must:** Be careful about heading hierarchy and placement of alerts. The analogy I lean on for heading structure is 9th grade English class, when you're writing an outline for an essay. Each topic or major section of the page is an H2, and each time you would indent your outline (sub-section, sub-sub-section, etc) you increase your heading level (H3, H4, etc).
   
   With no alerts, the contact information page is structured as:
   H1 - Contact information
      H2 - Addresses
         H3 - Mailing
         H3 - Home
      H2 - Phone numbers
         H3 - Home
         H3 - Work
         H3 - Mobile
      H2 - Email addresses
         H3 - Contact email
         H3 - Sign in email
   
   When you add in an alert with a heading ("Verify your email address to access VA.gov tools and features"), you have to decide where to insert it in the existing outline, ie. what existing section/sub-section/etc is its best and most relevant parent, and does it make sense alongside any siblings? And that informs what the right heading level should be.
   
   Note that headings can be styled however you'd like such that visual presentation makes sense --- so an alert heading can be an H4 but larger text than an H3, for example. But the heading level itself always has to match the structure of the content.
- [ ] **Should:** In the [verification email](https://www.figma.com/design/bFdl7MEIda4ExZIQuot84r/Profile---Contact-Information?node-id=1755-10340&t=XSXf9ftEhGAIFAtm-4), I would recommend adding explicit instructions for what to do if you receive the email in error. Hopefully that doesn't happen! But if you get a verification email that's not intended for you because a Veteran who's not you typo'd their email address, it's unclear from the current message if it's your responsibility to contact VA or if you should just ignore it or if there's something else you should do. That lack of clarity can pose cognitive load and anxiety concerns --- admittedly, not for our intended audience, but for someone who might still unintentionally interact with the email.
- [ ] **Consider:** After a verification message has been sent, consider including content to help set expectations --- "It may take a few minutes for you to receive the email" or similar, with instructions on what to do if the user believes enough time has passed they haven't received it. That kind of guidance may seem obvious but it can help with reducing anxiety and cognitive load (speaking as an anxious person).
- [ ] **Coding note:** If any background process ends up running such that the full mailbox and invalid email address alerts appear on screen without requiring a page reload, note that you'll need to use an [aria-live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-live) or something similar for those messages. Mentioning that here so that it's on your radar to talk to me about when you come back for a midpoint review!

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
